Don't install test files, i.e. don't include them in the package.

--- a/setup.cfg
+++ b/setup.cfg
@@ -44,9 +44,6 @@
 	weasyprint.css.validation
 	weasyprint.formatting_structure
 	weasyprint.layout
-	weasyprint.tests
-	weasyprint.tests.test_draw
-	weasyprint.tests.test_layout
 	weasyprint.tools
 zip_safe = false
 setup_requires = pytest-runner
